### PR TITLE
desktop version added

### PR DESCRIPTION
**Pull request by Nick-Shcherban**

### CODE REVIEW

File: styles/styles.css [hunk 1]
```diff
@@ -48,32 +62,79 @@ body {
     font-weight: 700;
     cursor: pointer;
 }
+.more_about_company:hover {
```
Comment (velmyk) [inline styles/styles.css]: It's better to use `-` instead of `_` for class names
`.more-about-company:hover`

File: styles/styles.css [hunk 2]
```diff
@@ -232,8 +302,4 @@ x
     margin-left: 20px;
 }
 
-@media screen and (orientation: portrait) {
-
-}
-
-
+@media screen and (orientation: portrait) {}
```
Comment (velmyk) [inline styles/styles.css:306]: Remove unused code

File: styles/styles.css [hunk 3]
```diff
@@ -176,8 +246,8 @@ body {
     line-height: 28px;
     color: #2B292D;
 }
-x
-.logo {
+
+x .logo {
```
Comment (velmyk) [inline styles/styles.css]: what is `x`?

**Review by velmyk — APPROVED** [review]

File: styles/styles.css (hunk 3 above)
Comment (Nick-Shcherban) [reply styles/styles.css]: deleted

File: styles/styles.css (hunk 2 above)
Comment (Nick-Shcherban) [reply styles/styles.css:306]: done

File: styles/styles.css (hunk 1 above)
Comment (Nick-Shcherban) [reply styles/styles.css]: done

**Pull request merged**